src/backend/binaryen/glue/transform.ts

```diff
@@ -172,10 +172,14 @@ export const baseStructType = initStructType(
 
 /** TS Function ${${}, funcref}*/
 export const builtinFunctionType = initStructType(
```

Comment:
Seems it's better to be named as "builtinClosureType"?

Reply:
Yes, I will rename it